### PR TITLE
Add gfix -icu to release notes, update gfix docs to Firebird 3

### PR DESCRIPTION
**Pull request by mrotteveel**

### CODE REVIEW

**Comment by MartinKoeditz** [comment]
I think, we can merge it. Who will do it?

**Comment by dyemanov** [comment]
I have no objections either. Plan to merge it later today.

**Comment by mrotteveel** [comment]
I used the merge request to ask Alex for feedback on firebird-devel, otherwise I would have committed it directly.

**Comment by dyemanov** [comment]
OK, but you could assign Alex as a reviewer in this PR instead of posting to fb-devel...

**Comment by mrotteveel** [comment]
@dyemanov Done, though my intent was more for an informal request for feedback, not a formal review.

File: src/docs/asciidoc/en/firebirddocs/gfix/firebird-gfix.adoc [hunk 1]
```diff
@@ -1445,13 +1470,45 @@ If you set the database to `async` mode (forced writes disabled) then it is poss
...
+
+NOTE: The `-{wj}nol[inger] option was introduced in Firebird 3.
+
+[[gfix-icu]]
+== Rebuild ICU dependencies
+
+Firebird uses the ICU library for some of its collations, and for indices on fields using those collations.
+When moving a database between different Firebird installations -- without using backup and restore -- and the destination Firebird installation uses a different ICU version, querying on fields with those collations or using those indices will fail.
+Before Firebird 3, the only option was to backup and restore the database.
+
```
Comment (AlexPeshkoff) [inline src/docs/asciidoc/en/firebirddocs/gfix/firebird-gfix.adoc:1498]: For linux need to use -icu switch may also take place after regular OS upgrade which may change version of system ICU library. Also you may mention that the switch is 'safe' - if no action actually need to be taken gfix will do nothing and exit immediately.

File: src/docs/asciidoc/en/rlsnotes/rlsnotes30/_utilities-30.adoc [hunk 2]
```diff
@@ -285,6 +285,13 @@ _gfix_ has a new switch `-NoLinger` to provide a one-off override to the `LINGER
 
 For information regarding `LINGER`, see <<rnfb30-ddl-enhance-linger,the write-up in the DDL chapter>>.
 
+[[rnfb30-util-gfix-icu]]
+=== -icu switch
+
+_gfix_ has a new switch `-icu` to update ICU-dependent collations and rebuild dependent indices.
+
+This can be used to make indices usable again when moving databases -- without backup and restore -- between Firebird installations using different ICU versions, or when the ICU version used by a Firebird install is updated.
```
Comment (AlexPeshkoff) [inline src/docs/asciidoc/en/rlsnotes/rlsnotes30/_utilities-30.adoc:293]: Why "by a Firebird install"? Install itself does not use ICU. Saying "when the ICU version used by a Firebird is updated" appears better.

**Review by AlexPeshkoff — CHANGES_REQUESTED** [review]

File: src/docs/asciidoc/en/firebirddocs/gfix/firebird-gfix.adoc (hunk 1 above)
Comment (mrotteveel) [reply src/docs/asciidoc/en/firebirddocs/gfix/firebird-gfix.adoc:1498]: Thanks, will update later this week

File: src/docs/asciidoc/en/rlsnotes/rlsnotes30/_utilities-30.adoc (hunk 2 above)
Comment (mrotteveel) [reply src/docs/asciidoc/en/rlsnotes/rlsnotes30/_utilities-30.adoc:293]: With "Firebird install" I meant an installed Firebird instance, not the installation process. I'll tweak the wording.

**Pull request merged**